src/xdocs/checks/header/regexpheader.xml

```diff
@@ -133,7 +122,7 @@ line 6: ^\W*$
               <td>Specify the name of the file containing the required header.</td>
               <td><a href="../../property_types.html#URI">URI</a></td>
               <td><code>null</code></td>
-              <td>3.2</td>
+              <td>4.4</td>
```

Comment:
Same.

Reply:
all since are reverted. map is updated with values from xdoc